reference/performance/execution-plan-bind.md

```diff
@@ -99,7 +99,41 @@ set global tidb_evolve_plan_baselines = on;
 
 The default value of `tidb_evolve_plan_baselines` is `off`.
 
-After the automatic binding evolution feature is enabled, if the optimal execution plan selected by the optimizer is not among the bound execution plans, the optimizer marks the plan as an execution plan that waits for verification. Every `bind-info-lease` (the default value is `3s`), an execution plan to be verified is selected and compared with a bound execution plan with the least cost, in terms of the actual execution time. If the plan to be verified is better, it is marked as a usable binding.
+> **Note:**
+>
+> The global variable doesn't take effect in the current session. It only takes effect in a newly created session. To enable automatic binding evolution in the current session, change the key word `global` to `session`.
+
+After the automatic binding evolution feature is enabled, if the optimal execution plan selected by the optimizer is not among the bound execution plans, the optimizer marks the plan as an execution plan that waits for verification. Every `bind-info-lease` (the default value is `3s`), an execution plan to be verified is selected and compared with a bound execution plan with the least cost, in terms of the actual execution time. If the plan to be verified is better, it is marked as a usable binding. The following use case describes the preceding process.
+
+Let table `t` be defined as:
+
+{{< copyable "sql" >}}
+
+```sql
+create table t(a int, b int, key(a), key(b));
+```
+
+Perform the following query on table `t`:
+
+{{< copyable "sql" >}}
+
+```sql
+select * from t where a < 100 and b < 100;
+```
+
+Rows in the table that satisfy condition `a < 100` are rare. However, for some reasons, the optimizer mistakenly chooses the full table scan instead of the optimal execution plan that uses index `a`. Users can use the following statement to create a binding first:
```

Comment:
```suggestion
In the table defined above, few rows meet the `a < 100` condition. However, for some reason, the optimizer chooses the full table scan instead of the optimal execution plan that uses index `a`. You can first use the following statement to create a binding:
```